styles/nav.css

```diff
@@ -0,0 +1,74 @@
+/* Header */
+
+header {
+    padding: calc(13rem / 16) 0;
+}
+
+/* Nav */
+
+:root {
+    --modal-header-padding: 0 1rem;
+}
+
+@media (min-width: 992px) {
+    :root {
+        --modal-header-padding: 0 calc(24rem / 16);
+    }
+}
+
+.modal-header {
+    padding: var(--modal-header-padding);
+    height: calc(72rem / 16); /* Match header height */
+}
+
+.navbar {
+    --bs-navbar-padding-y: 0;
+}
+
+.navbar-toggler-icon {
+    width: calc(32rem / 16);
+    height: calc(24rem / 16);
+    background-image: url('data:image/svg+xml,<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg"><rect y="4" width="32" height="2" fill="white"/><rect y="11" width="32" height="2" fill="white"/><rect y="18" width="32" height="2" fill="white"/></svg>');
+}
+
+.navbar-toggler-icon:hover {
+    background-image: url('data:image/svg+xml,<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg"><rect y="4" width="32" height="2" fill="%23878787"/><rect y="11" width="32" height="2" fill="%23878787"/><rect y="18" width="32" height="2" fill="%23878787"/></svg>');
+}
+
+.btn-close {
+    color: var(--white);
+    background: transparent url('data:image/svg+xml,<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(%23clip0_1358_20250)"><rect x="26.2341" y="28.0208" width="32" height="2" transform="rotate(-135 26.2341 28.0208)" fill="white"/><rect x="3.60669" y="26.6066" width="32" height="2" transform="rotate(-45 3.60669 26.6066)" fill="white"/></g><defs><clipPath id="clip0_1358_20250"><rect width="32" height="32" fill="white"/></clipPath></defs></svg>') center/2rem auto no-repeat;
+    width: 2rem;
+    height: 2rem;
+}
+
+.btn-close:hover {
+    background: transparent url('data:image/svg+xml,<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(%23clip0_1358_20250)"><rect x="26.2341" y="28.0208" width="32" height="2" transform="rotate(-135 26.2341 28.0208)" fill="%23878787"/><rect x="3.60669" y="26.6066" width="32" height="2" transform="rotate(-45 3.60669 26.6066)" fill="%23878787"/></g><defs><clipPath id="clip0_1358_20250"><rect width="32" height="32" fill="%23878787"/></clipPath></defs></svg>') center/2rem auto no-repeat;
+}
```

Comment:
I used this site https://www.svgbackgrounds.com/tools/svg-to-css/ to convert the SVG files from Figma into CSS data:images. Then, for the hover color change, I changed `fill="white"` to `fill="%23878787"`.

Reply:
Would the same `mask-image` approach from #121 work here and be simpler?

Reply:
Just a personal preference - but I actually prefer this approach -- technically this approach is compatible with more browsers. This approach doesn't work for #121 b/c the SVG is already encoded and the code doesn't have an easy `fill` value to change.